### PR TITLE
Update travis dist to xenial

### PR DESCRIPTION
**Pull request by kloon**
This PR updates the Travis build environment to use xenial instead of trusty which is quite old. Since we now have PHP 5.6 as a minimum and xenial supports PHP 5.6+ this should be fine. It will also allow us to also skip issues with certain packages being outdated and not support anymore on trusty since it is end of standard support already.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -61,6 +66,7 @@ before_script:
       bash tests/bin/install.sh woocommerce_test root '' localhost $WP_VERSION
       composer global require "phpunit/phpunit=5.7.*|7.5.*"
     fi
+  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
```
Comment (rodrigoprimo) [inline .travis.yml]: For future reference, could you please provide more details about what this command is doing and why it is needed? Thanks.

**Review by rodrigoprimo — APPROVED** [review]
LGTM

Left a comment asking for more details on a change just for future reference.

File: .travis.yml [hunk 2]
```diff
@@ -31,7 +36,7 @@ matrix:
     php: 7.2
     env: WP_VERSION=latest WP_MULTISITE=0 RUN_E2E=1
     addons:
-      chrome: stable
+      chrome: beta
```
Comment (rodrigoprimo) [inline .travis.yml]: Why do we need to run Chrome beta? Is this the change that fixed the e2e tests? Asking just to so we have this change documented somewhere.

Comment (kloon) [reply .travis.yml]: Yes, there is a bug in webdriver shipped with chrome stable that is the causing this. I see now harm in using beta though, might help us pick up browser changes earlier as well.

File: .travis.yml (hunk 1 above)
Comment (kloon) [reply .travis.yml]: Will add a comment

**Pull request merged**